src/HatchIntakeSubsystem.cpp

```diff
@@ -26,32 +26,32 @@ void HatchIntakeSubsystem::robotInit(){
     m_hatchActuator.SetSensorPhase(true);
 }
 
-void HatchIntakeSubsystem::teleopInit(){
+void HatchIntakeSubsystem::teleopInit() {
    
 }
 
-void HatchIntakeSubsystem::teleop(){
+void HatchIntakeSubsystem::teleop() {
     // When the right trigger is pressed the intake actuates down and spins the intake wheels until it gets
     // a hatch then it actuates up.
-    if (operatorJoystick->GetRisingEdge(CORE::COREJoystick::JoystickButton::RIGHT_TRIGGER)){
-        if(m_hatchActuator.GetSelectedSensorPosition() >= m_actuatorTopLimit.Get() - m_hatchActuatorDeadband.Get()){
+    if (operatorJoystick->GetRisingEdge(CORE::COREJoystick::JoystickButton::RIGHT_TRIGGER)) {
+        if(m_hatchActuator.GetSelectedSensorPosition() >= m_actuatorTopLimit.Get() - m_hatchActuatorDeadband.Get()) {
             SetHatchActuatorDown();
         }
-        if(m_hatchActuator.GetSelectedSensorPosition() <= m_actuatorBottomLimit.Get() + m_hatchActuatorDeadband.Get()){
-            if(GetHatchState()){
+        if(m_hatchActuator.GetSelectedSensorPosition() <= m_actuatorBottomLimit.Get() + m_hatchActuatorDeadband.Get()) {
+            if(GetHatchState()) {
                 SetHatchActuatorUp();
             }else{
                 SetIntake(true, !GetHatchState());
             }
         }
     }
     // When the right button is pressed, the intake actuates down and spits out the hatch and actuates up.
-    if (operatorJoystick->GetRisingEdge(CORE::COREJoystick::JoystickButton::RIGHT_BUTTON)){
-       if(m_hatchActuator.GetSelectedSensorPosition() >= m_actuatorTopLimit.Get() - m_hatchActuatorDeadband.Get()){
+    if (operatorJoystick->GetRisingEdge(CORE::COREJoystick::JoystickButton::RIGHT_BUTTON)) {
```

Comment:
You don't want to be using GetRisingEdge, because you want this to be looped through for as long as you are pressing the button.